ergast_py/models/time.py

```diff
@@ -0,0 +1,28 @@
+""" Time class """
+
+import datetime
+from dataclasses import dataclass
+
+
+@dataclass
+class Time:
+    """
+    Representation of a Formula One Team
+
+    Time may contain:
+        millis: datetime.time
+        time: String
```

Comment:
The naming of `time` is fairly confusing. I'm aware that that's the name provided by the Ergast API, I guess it's difficult given the nature of it either being the 1st place time, or the 2nd-20th place differences.

I think for the moment lets keep it named as time, as it's what Ergast reports.